CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 For official release notes for Docker Engine CE and Docker Engine EE, visit the
 [release notes page](https://docs.docker.com/engine/release-notes/).
 
-## 19.03.0 (2019-06-DD)
+## 19.03.0 (2019-06-17)
```

Comment:
Fixes:

- build: add SSH agent socket forwarder is not new in 19.03 
- builder-next: update buildkit to v0.5.0 <- not the last version
- cache-from <- put to a separate line

New things:
- BuildKit based builds use Dockerfile frontend version `docker/dockerfile:1.1` by default
- BuildKit based builds do not rely on an external image anymore for COPY/ADD operations
- BuildKit based builds will first look for a Dockerfile specific dockerignore file (eg. Dockerfile.dockerignore) for ignored paths
- BuildKit based builds automatically detect if process execution is possible for x86, arm and arm64 binaries.

Reply:
done